internal/jsonutil/graphql.go

```diff
@@ -132,10 +134,10 @@ func (d *decoder) decode() error {
 				return fmt.Errorf("struct field for %q doesn't exist in any of %v places to unmarshal", key, len(d.vs))
 			}
 
-			if rawMessage {
+			if rawMessage || isScalar {
 				// Read the next complete object from the json stream
 				var data json.RawMessage
-				d.tokenizer.Decode(&data)
+				_ = d.tokenizer.Decode(&data)
```

Comment:
Should we handle this a decoding error here?

Reply:
I think we should handle the decoding error.